source/extensions/http/stateful_session/cookie/cookie.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
+
+package envoy;
+
+message Cookie {
+  string address = 1;
+  uint32 expires = 2;
+}
```

Comment:
could we just add this `Cookie` to the `api/envoy/extensions/http/stateful_session/cookie/cookie.proto`?

Reply:
I think that `api` directory is for configuration. For representation of internal structures in protobufs, the proto definition is placed in source directory. Please see https://github.com/envoyproxy/envoy/blob/main/source/server/hot_restart.proto.

Reply:
reasonable and right.